src/main/resources/application.properties

```diff
@@ -5,4 +5,7 @@ spring.datasource.password=${DB_PASSWORD}
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
 spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
+jwt.expiration=${jwt_expiration}
+jwt.secret=${jwt_secret}
+
 server.port=8081
```

Comment:
can we change the port back to the default 8080?

Reply:
Will try. It was old problem resolving, didnt touch it atm.